pg_diffix--0.0.1.sql

```diff
@@ -34,7 +34,8 @@ LANGUAGE SQL
 AS $$
   SELECT objtype, objname, label
   FROM pg_seclabels
-  WHERE provider = 'pg_diffix';
+  WHERE provider = 'pg_diffix'
+  ORDER BY objtype, objname;
```

Comment:
Not necessary to do this for the sake of testing. We can filter some known subset to get a deterministic order. See `validation.sql` which filters them.

Reply:
> And does filtering itself guarantee stable order?

If the tests run then it's fine. The result order is possibly related to insertion order, and we always mark tables before columns. Ordering is fine but it's not necessary in my opinion.